CHANGES.md

```diff
@@ -5,8 +5,13 @@ all changes see the [closed
 milestones](https://github.com/cylc/cylc-uiserver/milestones?state=closed) for
 each release.
 
+<!-- The topmost release date is automatically updated by GitHub Actions. When
+creating a new release entry be sure to copy & paste the span tag with the
+`actions:bind` attribute, which is used by a regex to find the text to be
+updated. Only the first match gets replaced, so it's fine to leave the old
+ones in. -->
 -------------------------------------------------------------------------------
-## __cylc-uiserver-0.4 (2020-??-??)__
+## __cylc-uiserver-0.4.0 (<span actions:bind='release-date'>2021-04-?</span>)__
 
 ### Enhancements
```

Comment:
Not related to this change, but I noticed on GitHub UI, having two paragraphs, one after the other without any text, results in the second paragraph being displayed "raw", without formatting.

![image](https://user-images.githubusercontent.com/304786/113933649-f6c36580-9848-11eb-9e34-ed4190aa3472.png)

The other entries have a single paragraph like "This is the release 1.2.bla".

Reply:
Actually it was due to a non-breaking space after `###`